commit 2e3ebb9066b31e01f05c8a8d684b8998e8d9a6c4 (refs/changes/18/818/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-31T11:31:32-07:00 (2 years, 11 months ago)
    
    TOOLS-1594 headnode-joyent build failed with "mget: AssertionError: undefined (object) is required"

diff --git a/package.json b/package.json
index 8296a56..f72a187 100644
--- a/package.json
+++ b/package.json
@@ -1,11 +1,11 @@
 {
   "name": "mountain-gorilla",
   "description": "Build SDC components. Eat bamboo.",
-  "version": "2.0.0",
+  "version": "2.0.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-    "manta": "3.0.0",
+    "manta": "3.1.3",
     "smartdc": "7.3.0",
     "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#38332b1"
   },
